core/src/main/java/org/mskcc/cbio/portal/util/GeneticProfileReader.java

```diff
@@ -117,11 +117,16 @@ public static GeneticProfile loadGeneticProfile(File file) throws IOException, D
     	if (geneticProfile.getGeneticAlterationType() == GeneticAlterationType.GENESET_SCORE) {
             validateGenesetProfile(geneticProfile, file);
         }
+        
+        if (geneticProfile.getGeneticAlterationType() == GeneticAlterationType.TREATMENT) {
+            validateTreatment(geneticProfile, file);
```

Comment:
Rename `validateTreatment` to `validateTreatmentResponse` (I know this is not related to your code changes).